### ISSUE TITLE
Dubs Skylights 0% Passthrough issue, with troubleshooting

### ISSUE THREAD

**Issue by Kayedon** [jptrrs/OpenTheWindows#9]
I get the 50% indoor light bug with Open the Windows! loaded in a 300+ mod save when switching from my main colony, to Set Up Camp, and back. Remove the mod (and all my windows), issue goes away. 

Troubleshooting modlist in order: Harmony, Core, HugsLib, Dubs Skylights, [JPT] Open The Windows, [SYR] Set Up Camp.

Started new colony, god-mode built a little hut with a skylight. Confirmed 90% passthrough in 100% light environment. Made a caravan, set up camp. Switched back, light passthrough is still 90%. Save while at original colony and reload, repeat test of switching between colonies. Passthrough still 90%. **Save and reload from Set Up Camp colony**, passthrough is 0%.

Now here's where reproduction gets tricky. On my current save, once I switch to my side colony, the main one gets 0% passthrough until I save and reload a game while in the main colony, up until I switch back to my camp. I cannot reproduce that reliably - sometimes it gets "stuck" regardless of where the save is loaded from, sometimes it just works flawlessly provided the save (or autosave) was made at the main colony. Restarting the game seems to have no effect on which scenario I get.

Now, maybe the issue is with Set Up Camp? Nope. Tested without it, enabling 2 settlements. Same scenario - load a save from the 2nd colony, switch to first, skylights are set to 0%. Sometimes get stuck, sometimes don't. 

Removed Open The Windows, reloaded my 2 colony save - now the only mods are Harmony, Core, HugsLib, Dubs Skylights. Light is always 100% through the skylight in broad day light.

Tested setting Open The Windows light pass-through to 100% and 0%, both had same result.

I'm more than happy to assist in testing solutions. I think a "turn off this feature" toggle before the slider may be a good place to start for users to have access to while fixing this if possible?

**Comment by jptrrs** [comment]
Ok, thanks for bringing this up and testing different situations, Kayedon. Only thing that's missing is the game log for when there is an issue (you know: Ctrl+F12).
So, from what I understood, it ony happens after the game is reloaded and it only affects Dubs Skylights, is that right?

**Comment by Kayedon** [comment]
Laugh at me mate, I completely forgot about my log. Here you are, but I re-did the test a bit. Turns out the issue is not with Dubs Skylights, that's just where I noticed it the most. It's with anything this mod adds light passthrough to, including the windows.

Load order: Harmony, Core, HugsLib, [JPT] Open The Windows.

Same test, load a new game, spawn a hut with some windows in my main colony, set up a new settlement, make a save in the 2nd colony, reload, switch back to first, light passthrough is 0%. 

https://gist.github.com/d19bdf52288a974a37dba412e27eb6be
